### PR TITLE
adding twilio connector entry with description fields

### PR DESCRIPTION
**Pull request by iamjamilspain**
### Description
Added Twilio Connector to the datasources list

#### Related issues

- connect to <link_to_referenced_issue>

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by dhmlau** [comment]
@jamilhassanspain, thanks for the PR.  Could you please fix the commit linter error?  Same as the other PR you have created, you can follow this instruction to amend the commit message: https://help.github.com/articles/changing-a-commit-message/.

```
**************************************************
**
**  Linting commit logs
**
**  1 problems found:
**    cd5f213 - adding twilio connector entry with description fiel: First line should be 50 characters or less (saw 53)
**
**************************************************
```

FYI - as a reference, please see our [commit message guideline](https://loopback.io/doc/en/contrib/git-commit-messages.html) for details.  Thanks!

File: available-connectors.json [hunk 1]
```diff
@@ -668,6 +668,30 @@
     },
     "supportedByStrongLoop": false
   },
+  {
+    "name": "twilio",
+    "description": "Twilio Connector",
+    "baseModel": "Model",
```
Comment (jannyHou) [inline available-connectors.json]: I am not very familiar with twilio, usually if the data is persistently stored on the back end, the `baseModel` should be `PersistedModel`.

(Ignore my comment if you understand the difference between the 2 base models and choose `Model` with a good reason.)

**Comment by emonddr** [comment]
Is there a `mocha test` that can be included to illustrate the purpose/goal of this PR? Thx.

**Comment by dhmlau** [comment]
Thanks @iamjamilspain for your contribution.  Your PR has landed 🎉

**Pull request merged**